include/deal.II/lac/precondition.h

```diff
@@ -461,10 +461,10 @@ class PreconditionRelaxation : public Subscriptor
  *
  * //...initialize and build A
  *
- *     // Define and initialize preconditioner
+ * // Define and initialize preconditioner:
  *
  * PreconditionJacobi<SparseMatrix<double> > precondition;
- * precondition.initialize (A, .6);
+ * precondition.initialize (A, typename PreconditionJacobi<SparseMatrix<double> >::AdditionalData(.6));
```

Comment:
> I think it even works without this change because your constructor is not explicit and will convert the double automatically.

correct, it will. But I also find it clearer this way as when you look at the functions, there is no `initialize` which takes double as the second argument.